eds4jinja2/adapters/local_sparql_ds.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+
+# remote_sparql_ds.py
+# Date:  07/08/2020
+# Author: Eugeniu Costetchi
+# Email: costezki.eugen@gmail.com
+
+
+import pandas as pd
+import rdflib
+
+from eds4jinja2.adapters.base_data_source import DataSource, UnsupportedRepresentation
+
+DEFAULT_ENCODING = 'utf-8'
+
+
+class LocalSPARQLDataSource(DataSource):
+    """
+        Fetches data from SPARQL local datasource. Can be used with a SPARQL query.
```

Comment:
"""Accesses a local RDF file and provides the possibility to fetch data from it by SPARQL queries. """